oak_dice/src/evidence.rs

```diff
@@ -114,7 +114,7 @@ impl RootLayerEvidence {
 static_assertions::assert_eq_size!([u8; 2320], RootLayerEvidence);
 
 /// Attestation evidence for non-root layers.
-#[derive(AsBytes, FromZeroes, FromBytes)]
+#[derive(AsBytes, FromZeroes, FromBytes, Clone, Copy)]
```

Comment:
Implementing this as applications receiving AttestationEvidence will likely want to clone it (based on current usage)

Reply:
I agree with `Clone`, but not sure `Copy` is needed as it might lead to accidental cloning via assignment.

Reply:
oki, removed it